### ISSUE TITLE
Is there a way to find indices of the points which are sampled with 'farthest data sampling' method?

### ISSUE THREAD

**Issue by ashar6194** [charlesq34/pointnet2#32]
Or how exactly should we sample the corresponding label of each single point?

**Comment by oafolabi** [comment]
Yes, you may have to get the tensor that does that operation. For example, you may do something similar to the following:

You may restore the session and  get the placeholders like this : 
```
sess = tf.Session(config=config)

init = tf.global_variables_initializer()
sess.run(init)

saver = tf.train.import_meta_graph(META_GRAPH)
saver.restore(sess, tf.train.latest_checkpoint(LOG_DIR))

# We can now access the default graph where all our metadata has been loaded
graph = tf.get_default_graph()

# tmp = graph.get_operations()

# tmp = [n.name for n in graph.as_graph_def().node]

# print tmp

 
# placeholders we need
farthest_points_pl = graph.get_tensor_by_name('layer1/FarthestPointSample:0')
pointclouds_pl = graph.get_tensor_by_name('Placeholder:0')
pred_pl = graph.get_tensor_by_name('fc2/BiasAdd:0')

is_training_pl = graph.get_tensor_by_name('Placeholder_3:0')
```
META_GRAPH and LOG_DIR are directory paths you define. 

Then run a forward pass: 
```
TEST_DATASET = scannet_dataset.ScannetDatasetWholeScene(...)
batch_data, batch_label, batch_smpw = TEST_DATASET[scene_idx]
feed_dict = {pointclouds_pl: batch_data,
            is_training_pl: False}
farthest_points, pred_val = sess.run([farthest_points_pl,pred_pl], feed_dict=feed_dict)
```

farthest_points should give you something close to what you're looking for.  Please see 
https://github.com/charlesq34/pointnet2/blob/7961e26e31d0ba5a72020635cee03aac5d0e754a/utils/pointnet_util.py#L87

and it's containing file for more info. 

Hopefully that helps.
